pages-e2e/features/pagesGetRequestContext/assets/wrangler.toml

```diff
@@ -0,0 +1,6 @@
+[vars]
+MY_TOML_VAR = "my var from wrangler.toml"
+
+[[kv_namespaces]]
+binding = "MY_TOML_KV"
+id = "???"
```

Comment:
??? ?

Reply:
I was just wondering if this was on purpose... like you were not sure what to put here.

Reply:
ah ok, well let's put `x`es here just to make it clearer and consistent with [what we do in c3/wrangler](https://github.com/cloudflare/workers-sdk/blob/7412ce7e6674c37b4263319bb255064c5702321f/packages/create-cloudflare/templates/hello-world/js/wrangler.toml#L15) 🙂